testing-overhead-jmh/src/jmh/java/io/opentelemetry/javaagent/benchmark/ServletWithAgentBenchmark.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.benchmark;
+
+import org.openjdk.jmh.annotations.Fork;
+
+@Fork(
+    jvmArgsPrepend = {
+      "-javaagent:build/agent/opentelemetry-javaagent-all.jar",
```

Comment:
Is it better to set these in the Gradle file instead of in the annotation? Then you wouldn't need the copyAgent stuff

Reply:
I gave it a try, I think it's better now(?), at least it matches smoke-tests, which is good